client/js/store/procedure/NewProcedure.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * (c) 2010-present DEMOS E-Partizipation GmbH.
+ *
+ * This file is part of the package demosplan,
+ * for more information see the license file.
+ *
+ * All rights reserved
+ */
+
+const NewProcedure = {
+  namespaced: true,
+
+  name: 'newProcedure',
+
+  state: {
+    requireField: true
+  },
+
+  mutations: {
+    /**
+     *
+     * @param state {Object}
+     * @param value {Boolean} Sets required form field dynamically to true/false.
```

Comment:
Nitpick: Maybe we could use this comment as a doc block for the mutation (and remove it from the parameter), adjusting it like so:
"Sets `required` attribute of a form field dynamically to true/false if the attribute is bound to `requireField`"?

Reply:
Thanks, I think this is a good idea, done in [b90261c](https://github.com/demos-europe/demosplan-core/pull/877/commits/b90261cccb01b164946b8d07bd379c2fc1bc42b8)